server.js

```diff
@@ -21,3 +22,12 @@ app.listen(PORT, function(){
 //     headers: {Authorization: `token ${process.env.GITHUB_TOKEN}`}
 //   }))(request, response);
 // }
+function proxyGitHub(request, response) {
```

Comment:
For this part of the functionality, you do not need to do this much. Nor inside the server file. Here is a link to the finished code example sam walked through in class. This may help! 
https://github.com/codefellows/301-12-github-api/tree/finished